src/main/java/com/ldtteam/domumornamentum/block/vanilla/WallBlock.java

```diff
@@ -118,19 +114,6 @@ public void fillItemCategory(final @NotNull NonNullList<ItemStack> items)
         items.addAll(fillItemGroupCache);
     }
 
-    @Override
-    public void setPlacedBy(
```

Comment:
In some cases you remove it and in some cases you change it. What is different there?

Reply:
i removed the be data setter it's handled by vanilla, however few methods like doors are needed cuz of other-half blocks, or some of them applies TYPE which i have no idea where it comes from so i havent touched it